Make.defs

```diff
@@ -108,10 +108,6 @@ define REGISTER
 	$(Q) touch "$(BUILTIN_REGISTRY)$(DELIM).updated"
 endef
 
-define ARLOCK
-	$(Q) flock $1.lock $(call ARCHIVE, $1, $(2))
```

Comment:
Please don't forget to remove the flock from CI script and the document.

Reply:
Right. The documentation is on nuttx repo so I will add the change there.
As per removing flock from CI this is on the testing repo. I can do that once these two PRs are merged.